### ISSUE TITLE
wrong-exception-operation raised when concat tuple in except

### ISSUE THREAD

**Issue by rany2** [pylint-dev/pylint#9288]
### Bug description

Pylint wrong-exception-operation is raised when concatenating the tuple in except. 

Consider this file `a.py`:

```python
EXC = (ValueError,)

try:
	int("hi")
except (OSError,) + EXC as exc:
	print("caught it (exc={})".format(exc))

try:
	raise OSError("whatever")
except (OSError,) + EXC as exc:
	print("caught it (exc={})".format(exc))
```

Pylint complains with:

```
************* Module a
a.py:5:7: W0716: Invalid exception operation. Did you mean '((OSError, ), EXC)' instead? (wrong-exception-operation)
a.py:10:7: W0716: Invalid exception operation. Did you mean '((OSError, ), EXC)' instead? (wrong-exception-operation)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

when this case is valid.

### Command used

```shell
pylint a.py
```

### Pylint output

```python
************* Module a
a.py:5:7: W0716: Invalid exception operation. Did you mean '((OSError, ), EXC)' instead? (wrong-exception-operation)
a.py:10:7: W0716: Invalid exception operation. Did you mean '((OSError, ), EXC)' instead? (wrong-exception-operation)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

### Expected behavior

`wrong-exception-operation` shouldn't be raised

### Pylint version

```shell
pylint 3.0.2
astroid 3.0.1
Python 3.11.6 (main, Oct  2 2023, 13:45:54) [Clang 15.0.0 (clang-1500.0.40.1)]
```

**Comment by rany2** [comment]
For anyone experiencing this issue, instead of disabling this warning you could do something like:

```
>>> x=(1,2,3,4)
>>> (*x, 5, 6, 7)
(1, 2, 3, 4, 5, 6, 7)
```

I think it looks better as well, so maybe pylint should have some style warning suggesting:

```python3
EXC = (ValueError,)

try:
	int("hi")
except (*EXC, OSError) as exc:
	print("caught it (exc={})".format(exc))
```

for

```python3
EXC = (ValueError,)

try:
	int("hi")
except (OSError,) + EXC as exc:
	print("caught it (exc={})".format(exc))
```
